src/rust/mangastream/template/src/template.rs

```diff
@@ -400,7 +402,7 @@ impl MangaStreamSource {
 		} else {
 			for (at, page) in html.select(self.page_selector).array().enumerate() {
 				let page_node = page.as_node().expect("Failed to get page as node");
-				let page_url = urlencode(page_node.attr(self.page_url).read());
+				let page_url = if self.protocol { format!("{}{}", "https:", urlencode(page_node.attr(self.page_url).read())) } else { urlencode(page_node.attr(self.page_url).read()) } ;
```

Comment:
this shouldn't be all on one line, run `cargo fmt` and it'll format it for you correctly

Reply:
Done. Do you think it's worth it to have it run as part of a pre-commit linter?